### PR TITLE
SmtpClient doc correction for .net core. Fixes #4705

### LINKED ISSUE #4705: Documentation issue on SmtpClass page
# SmtpClass 

In  https://docs.microsoft.com/en-us/dotnet/api/system.net.mail.smtpclient?view=netcore-2.0 it says 

the description below is wrong and doesn't apply to Core.
```
You can use the application or machine configuration files to specify default host, port, and credentials values for all SmtpClient objects. For more information, see <mailSettings> Element (Network Settings)
```

In order to specify default host, port, and credentials values, we need to set the relevant properties on SmtpClient directly.

More info [here](https://github.com/dotnet/corefx/issues/25127)

cc: @mairaw

### PR DESCRIPTION
**Pull request by maryamariyan**
## Summary

SmtpClient doc correction for .net core. 

Fixes #4705

cc: @mairaw

### CODE REVIEW

File: xml/System.Net.Mail/SmtpClient.xml [hunk 1]
```diff
@@ -53,7 +53,7 @@
...
   
- You can use the application or machine configuration files to specify default host, port, and credentials values for all <xref:System.Net.Mail.SmtpClient> objects. For more information, see [&lt;mailSettings&gt; Element (Network Settings)](~/docs/framework/configure-apps/file-schema/network/mailsettings-element-network-settings.md).  
+ In .NET Framework you can use the application or machine configuration files to specify default host, port, and credentials values for all <xref:System.Net.Mail.SmtpClient> objects. For more information, see [&lt;mailSettings&gt; Element (Network Settings)](~/docs/framework/configure-apps/file-schema/network/mailsettings-element-network-settings.md). But in .NET Core we don't support setting defaults. As a workaround you need to set the relevant properties on SmtpClient directly.
```
Comment (rpetrusha) [inline xml/System.Net.Mail/SmtpClient.xml]: - comma after .NET Framework
- The .NET Core sentence should be rephrased to remove "But" and "we": ".NET Core does not support setting defaults. As a workaround, you must set the relevant properties on SmtpClient directly.  
- fence SmtpClient (\`SmtpClient\`)

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for adding information on .NET Core, @maryamariyan. I've made a couple of comments and suggestions.

**Comment by rpetrusha** [comment]
Incidentally, @maryamariyan, it wasn't necessary to open an issue, since this is a minor correction.

File: xml/System.Net.Mail/SmtpClient.xml (hunk 1 above)
Comment (maryamariyan) [reply xml/System.Net.Mail/SmtpClient.xml]: thanks @rpetrusha . How do I fence SmtpClient?
this way: `SmtpClient`
or 
this way: <xref:System.Net.Mail.SmtpClient>

**Comment by rpetrusha** [comment]
This looks good. Thanks for making the additional changes, @maryamariyan. We'll merge now.

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**